src/core/utils/RovingFocusGroup/fragments/RovingFocusRoot.tsx

```diff
@@ -18,6 +18,7 @@ type RovingFocusRootProps = {
     loop?: boolean;
     'aria-label'?: string;
     'aria-labelledby'?: string;
+    rovingFocusDisabled?: boolean;
```

Comment:
Should we call it  `disableTabIndexing`? 

I think `rovingFocusDisabled` prop for a RovingFocusGroup comp feels counter intuitive :)